boards/arm/atsamr21_xpro/atsamr21_xpro.dts

```diff
@@ -49,6 +49,13 @@
 	txpo = <0>;
 };
 
+&sercom1 {
+	status = "ok";
+	compatible = "atmel,sam0-i2c";
+	clock-frequency = <400000>;
```

Comment:
Use `I2C_BITRATE_FAST` from `include/dt-bindings/i2c/i2c.h` instead of hardcoding the number

Reply:
Ok.